plone/recipe/codeanalysis/__init__.py

```diff
@@ -77,6 +79,9 @@ def __init__(self, buildout, name, options):
         # CSS Lint
         self.options.setdefault('csslint', 'False')
         self.options.setdefault('csslint-bin', 'csslint')
+        # check-manifest
+        self.options.setdefault('check-manifest', 'True')
```

Comment:
Sure. I'm not against having that check here as well.

Am 28.04.2015 um 10:26 schrieb Gil Forcada Codinachs:
> In plone/recipe/codeanalysis/__init__.py
> <https://github.com/plone/plone.recipe.codeanalysis/pull/113#discussion_r29222641>:
> 
>> @@ -77,6 +79,9 @@ def __init__(self, buildout, name, options):
>>          # CSS Lint
>>          self.options.setdefault('csslint', 'False')
>>          self.options.setdefault('csslint-bin', 'csslint')
>> +        # check-manifest
>> +        self.options.setdefault('check-manifest', 'True')
> 
> @tisto <https://github.com/tisto> being |check-manifest| an external
> package I don't mind/care if zest.releaser or whatnot is /also/
> including it, but I do care that all our general QA checks that we want
> to do are here at p.r.codeanalysis.
> 
> For me p.r.codeanalsysis is basically the one-stop shop for anything
> regarding QA. So in short :+1: to keep it here :)
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/plone/plone.recipe.codeanalysis/pull/113/files#r29222641>.
>